### PR TITLE
Clang 13 complains about an unused variable

### PR DESCRIPTION
**Pull request by bold84**

### CODE REVIEW

**Comment by pepone** [comment]
Hi @bold84 ,

PR looks good, can you sign the contributor agreement, so we can merge this?

Cheers,
Jose

**Comment by bold84** [comment]
> Hi @bold84 ,
> 
> PR looks good, can you sign the contributor agreement, so we can merge this?
> 
> Cheers, Jose

Already done :-) 

Screen Shot 2022-03-29 at 15 15 42

File: cpp/src/slice2swift/SwiftUtil.cpp [hunk 1]
```diff
@@ -1523,19 +1523,14 @@ SwiftGenerator::writeMemberwiseInitializer(IceUtilInternal::Output& out,
                                            const DataMemberList& baseMembers,
                                            const DataMemberList& allMembers,
                                            const ContainedPtr& p,
-                                           bool local,
+                                           bool /*local*/,
```
Comment (pepone) [inline cpp/src/slice2swift/SwiftUtil.cpp:1526]: Would be better to remove this parameter, and fix the places where this is called.

Comment (bold84) [reply cpp/src/slice2swift/SwiftUtil.cpp:1526]: I didn't want to be too invasive :-P

I will have a look at it.

**Comment by bold84** [comment]
Builds on MacOS M1

Screen Shot 2022-03-29 at 22 06 23

**Review by pepone — APPROVED** [review]
Looks good, thanks for contributing to Ice

**Review by externl — APPROVED** [review]
Looks good to me!

**Pull request merged**